router/pkg/config/config.schema.json

```diff
@@ -935,6 +940,11 @@
                   "default": false,
                   "description": "Enable the collection of metrics for the GraphQL operation router caches. The default value is false."
                 },
+                "engine_stats" : {
+                  "type": "boolean",
+                  "default": false,
+                  "description": "Enable the collection of metrics for the Engine statistics. The default value is false."
```

Comment:
Maybe be a bit more verbose in what metrics we will enable and why they are useful.